codecov.yml

```diff
@@ -12,4 +10,6 @@ comment:
   branches: null
   behavior: default
   flags: null
-  paths: null
\ No newline at end of file
+  paths: null
+ignore:
+  - "examples/"
```

Comment:
CodeCov often rejects uploads; this probably unrelated but from logs it might get caught up on this example with "coverage" in its name


```
info - 2024-05-31 17:49:13,111 -- Found 2 coverage files to report
info - 2024-05-31 17:49:13,111 -- > /home/runner/work/openff-toolkit/openff-toolkit/coverage.xml
info - 2024-05-31 17:49:13,111 -- > /home/runner/work/openff-toolkit/openff-toolkit/examples/deprecated/check_dataset_parameter_coverage/check_parameter_coverage.ipynb